examples/default/versions.tf

```diff
@@ -0,0 +1,16 @@
+##############################################################################
+# Terraform Providers
+##############################################################################
+
+terraform {
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = ">=1.43.0"
```

Comment:
In examples, we tend to lock into the lowest version of the the range defined in the main module's `versions.tf` file to ensure the lowest version in the range continues to work since the examples run as part of the pr tests.
So I would recommend locking into `version = "1.43.0"` here.